lib/events/events.proto

```diff
@@ -32,6 +32,9 @@ message Metadata {
 message SessionMetadata {
     // SessionID is a unique UUID of the session.
     string SessionID = 1 [ (gogoproto.jsontag) = "sid" ];
+
+    // ClusterName identifies the originating teleport cluster
+    string ClusterName = 2 [ (gogoproto.jsontag) = "cluster_name" ];
```

Comment:
I'm curious whether you've considered placing this in a separate `ClusterMetadata` struct? I don't know if cluster name really belongs to the session metadata, seems like a separate "class" of information to me but I might be misunderstanding.

Reply:
What about `ServerMetadata`?